docs/source/_templates/autoapi/python/class.rst

```diff
@@ -44,7 +44,6 @@ This is currently not possible, see https://github.com/readthedocs/sphinx-autoap
    {% if visible_overview_methods %}
 
    .. admonition:: Methods
-      :class: admoition-methods
```

Comment:
If I checked correctly, this is nowhere used. However, it might be helpful in the future. So maybe fix the typo instead?

Reply:
Good point! Would be fine for me.